tests/std/tests/P0896R4_views_take/test.cpp

```diff
@@ -82,7 +82,7 @@ concept CanViewTake = requires(Rng&& r) {
 template <ranges::input_range Rng, ranges::random_access_range Expected>
 constexpr bool test_one(Rng&& rng, Expected&& expected) {
     using ranges::input_range, ranges::forward_range, ranges::bidirectional_range, ranges::random_access_range,
-        ranges::contiguous_range;
+        ranges::contiguous_range, ranges::borrowed_range;
     using ranges::take_view, ranges::common_range, ranges::enable_borrowed_range, ranges::iterator_t, ranges::prev,
         ranges::range, ranges::sentinel_t, ranges::sized_range;

```

Comment:
Nitpick: There are two lists of using-declarations here. The first list is for various "strengths" of ranges, while the second list is for other machinery. To be consistent with this organization, `ranges::borrowed_range` should be added to the "other machinery" list.

(Compare `P0896R4_views_drop` which is following the existing organization, although its lists occur in the opposite order.)